### DISCUSSION TITLE
Question developer mode

### DISCUSSION THREAD

**Discussion by adamigom** [magicbug/Cloudlog#1733] [Q&A] [answered]
I have a question that I have not found an answer to, what is developer mode and how to turn it off ?

![Screenshot_2022-10-31-00-00-36-42_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/117047000/198906198-c9254ba4-12cf-4723-95a3-e852138e3fa5.jpg)

**Comment by phl0** [comment, marked as answer]
See https://github.com/magicbug/Cloudlog/issues/1715#issuecomment-1288492483.

**Comment by adamigom** [comment]
Beautifully thank you for your help you can close the topic.

**Reply by phl0** [reply]
I can't. I guess you have to do that please.

**Comment by AndreasK79** [comment]
@magicbug why ship with this active in release?
